pyproject.toml

```diff
@@ -1,24 +1,38 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
 [tool.poetry]
 name = "hyperframe"
-version = "0.1.1"
+version = "0.1.2"
 description = ""
 authors = ["SamB <58937194+sam-lawrence@users.noreply.github.com>"]
 readme = "README.md"
+repository = "https://github.com/sam-lawrence/HyperFrame"
+classifiers = [
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3 :: Only",
+]
+packages = [{ include = "hyperframe" }, { include = "hyperframe/py.typed" }]
```

Comment:
Is this to stop it including `.github/workflows`??  And what does `hyperframe/py.typed` do?

Reply:
yep, it's a list of modules to include in the built distribution https://python-poetry.org/docs/pyproject/#packages

py.typed marks this package as a typed package